### ISSUE TITLE
Does fuse-box support require.js style amd?

### ISSUE THREAD

**Issue by rafde** [fuse-box/fuse-box#575]
I've been trying to get something like
```js
define([
  'mymodule',
  'mymodule2'
], function(mymodule, mymodule2) {
});
```
or
```js
    require([
      'uglymodule'
    ], function() {

      require([
        'uglymodule2'
      ]);
    });
```
or 
```js
define(function(require, exports, module) {
  var mySugarModule = require('mySugarModule');
  exports.stuff = 'good';
});
```
working and it's not panning out. It made me question if it's supported. I wanted confirmation.
If any of these are not support, I'll make a PR against https://github.com/fuse-box/fuse-box/blob/master/docs/faq.md to inform future surveyors about any of these not having support.
Thanks.

**Comment by nchanged** [comment]
Hi! Thanks for the issue. Yes we don't support it yet, However, there are 2 PR's pending on the issue.
https://github.com/fuse-box/fuse-box/pull/566
https://github.com/fuse-box/fuse-box/pull/567

It will be probably solved after the upcoming release

**Comment by rafde** [comment]
Cool. Something for me to test over the weekend.

**Comment by rafde** [comment]
Hmm... I noticed one of the issues closed and the PR has not been address, so I figured I probably didn't describe the problem well enough.
I was asking if it was possible for FuseBox to read the examples I provided.
I am working on a project that uses RequireJs in development and webpack for production bundling. Because of the large amount (+1000) of files,  webpack for development takes 40s to 1 minute to update a bundle. I can't update over 1000 files to use ES6 import all at once since it will be a regression test nightmare. Since FuseBox boast speed, I was looking forward to test it out. But after finding that it only supports import statement, I will have to keep looking into finding ways to speed up webpack bundling.
:man_shrugging:
